main.go

```diff
@@ -292,3 +293,12 @@ func addDependencies(_ context.Context, mgr ctrl.Manager, cfg config.Config, v v
 func minTlsDefault(cfg *tls.Config) {
 	cfg.MinVersion = defaultMinTLSVersion
 }
+
+func secureCipherSuite(cfg *tls.Config) {
```

Comment:
Could you please add a comment on this function and describe what it does? 

I personally don't know why all ciphers have to be specified.

Reply:
By description from the comments, it should be taken in safe cipher. But after i run the test, seems like manager just directly using the cipher configuration by tls version only. Instead of using the ciphers from tls.CipherSuites() which is safe from vulnerability.

Reply:
@pavolloffay do u have any more concerned, or may b do u have any more ideas to improve on this?

Reply:
Ok let me update the comments on the code and it will be more clear for reader/maintainer